src/utils/staking.js

```diff
@@ -295,21 +295,24 @@ export class Staking {
                 .filter((v) => v.indexOf('nfvalidator') === -1 && v.indexOf(networkId === 'mainnet' ? '.near' : '.m0') > -1)
         }
 
+        const account = await this.wallet.getAccount(this.wallet.accountId)
+
         return (await Promise.all(
             accountIds.map(async (account_id) => {
+                if (!account_id) return
```

Comment:
why would we have empty account_id in the list?

Reply:
I'll revert was from before, I thought validator_id was undefined